### PR TITLE
Stanza log

### PR DESCRIPTION
**Pull request by chrzaszcz**
- Update the docs after last change.
- Log stanzas received in escalus_connection.
- Get rid of the transport matching hack.

### CODE REVIEW

File: src/escalus_client.erl [hunk 1]
```diff
@@ -156,8 +154,3 @@ make_jid(Proplist) ->
     {resource, R} = lists:keyfind(resource, 1, Proplist),
     <<U/binary,"@",S/binary,"/",R/binary>>.
```
Comment (gadgetci) [inline src/escalus_client.erl]: According to **Elvis**:
> Missing space right "," on line 155

Comment (gadgetci) [inline src/escalus_client.erl]: According to **Elvis**:
> Missing space right "," on line 155

Comment (gadgetci) [inline src/escalus_client.erl]: According to **Elvis**:
> Missing space right "," on line 155

Comment (gadgetci) [inline src/escalus_client.erl]: According to **Elvis**:
> Missing space right "," on line 155

**Comment by chrzaszcz** [comment]
Previous merge from this branch was too quick :-(
Let me test it for a few days now.

**Pull request merged**